### PR TITLE
modify model-serving

### PR DESCRIPTION
**Pull request by harperjuanl**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -153,4 +153,5 @@ dmypy.json
 # Cython debug symbols
 cython_debug/
 
+
```
Comment (liuqi) [inline .gitignore]: Just an empty line change. Drop this?

File: docs/source/model-serving.rst [hunk 2]
```diff
@@ -11,19 +11,32 @@ Model Serving Basics
...
+about model serving as following,
+
+* Helping to standardize model serving across orgs with unified data plane and pre-built model servers
+* A single way to deploy, monitor inference services/server, and scale inference workload
+* Dramatically shortening time for data scientists to deploy models to production
 
 Overview
 ++++++++
 
-.. figure:: ./figs/model-serving-architecture.png
```
Comment (liuqi) [inline docs/source/model-serving.rst:19]: Remove this file from git, since it is not used anymore?

**Review by liuqi — CHANGES_REQUESTED** [review]
see comments inline. thanks.

File: docs/source/model-serving.rst (hunk 2 above)
Comment (harperjuanl) [reply docs/source/model-serving.rst:19]: remove msa figure

File: .gitignore (hunk 1 above)
Comment (harperjuanl) [reply .gitignore]: drop the empty line

**Review by harperjuanl — COMMENTED** [review]
modify the comments

**Pull request merged**